src/test/e2e/Helpers/Selectors.ts

```diff
@@ -102,4 +102,7 @@ module.exports = {
 
   // UnsubscribeConfirmation selectors
   panelTitle: '.govuk-panel__title',
+
+  //Daily Cause list
+  DailyCauseListTitle: 'h2.govuk-heading-l',
```

Comment:
we already have `.govuk-heading-l` in common, just need to use `CommonPageTitle`